docs/source/checks/tabular/model_evaluation/plot_calibration_score.py

```diff
@@ -28,30 +50,17 @@ def custom_formatwarning(msg, *args, **kwargs):
 #%%
 # Binary Classification
 # =====================
-# Load data
-# ---------
-# The dataset is the adult dataset which can be downloaded from the UCI machine
-# learning repository.
-#
-# Dua, D. and Graff, C. (2019). UCI Machine Learning Repository [http://archive.ics.uci.edu/ml].
-# Irvine, CA: University of California, School of Information and Computer Science.
-
-from urllib.request import urlopen
-
-from sklearn.preprocessing import LabelEncoder
-
-label_name = 'income'
 
 #%%
-
-from deepchecks.tabular import Dataset
+# Generate data & model
+# --------------------
```

Comment:
```suggestion
# -------------------------
```

(sphinx requires title underline to be at least the same length as text above, otherwise it triggers a warning)